packages/sources/iex-cloud-test/src/stock-utils.ts

```diff
@@ -0,0 +1,130 @@
+import { config } from './config'
+import { SingleNumberResultResponse } from '@chainlink/external-adapter-framework/util'
+import { InputParameters } from '@chainlink/external-adapter-framework/validation'
+
+export const inputParameters = {
+  base: {
+    aliases: ['from', 'coin', 'asset', 'symbol'],
+    description: 'The symbol to query',
+    required: true,
+    type: 'string',
+  },
+} satisfies InputParameters
+
+interface RequestParams {
+  base: string
+}
+
+interface ResponseSchema {
+  avgTotalVolume: number
+  calculationPrice: string
+  change: number
+  changePercent: number
+  close: number
+  closeSource: string
+  closeTime: number
+  companyName: string
+  currency: string
+  delayedPrice: number
+  delayedPriceTime: number
+  extendedChange: number
+  extendedChangePercent: number
+  extendedPrice: number
+  extendedPriceTime: number
+  high: number
+  highSource: string
+  highTime: number
+  iexAskPrice: number
+  iexAskSize: number
+  iexBidPrice: number
+  iexBidSize: number
+  iexClose: number
+  iexCloseTime: number
+  iexLastUpdated: number
+  iexMarketPercent: number
+  iexOpen: number
+  iexOpenTime: number
+  iexRealtimePrice: number
+  iexRealtimeSize: number
+  iexVolume: number
+  lastTradeTime: number
+  latestPrice: number
+  latestSource: string
+  latestTime: string
+  latestUpdate: number
+  latestVolume: number
+  low: number
+  lowSource: string
+  lowTime: number
+  marketCap: number
+  oddLotDelayedPrice: number
+  oddLotDelayedPriceTime: number
+  open: number
+  openTime: number
+  openSource: string
+  peRatio: number
+  previousClose: number
+  previousVolume: number
+  primaryExchange: string
+  symbol: string
+  volume: number
+  week52High: number
+  week52Low: number
+  ytdChange: number
+  isUSMarketOpen: boolean
+}
+
+export type StockEndpointTypes = {
+  Request: {
+    Params: RequestParams
+  }
+  Response: SingleNumberResultResponse
+  Settings: typeof config.settings
+  Provider: {
+    RequestBody: never
+    ResponseBody: ResponseSchema
+  }
+}
+
+export const buildHttpRequestBody = (params: RequestParams[], c: typeof config.settings) => {
+  return params.map((param) => {
+    return {
+      params,
```

Comment:
Same here. Since we don't batch we'll want 
```suggestion
      params: [param],
```